release-plz.toml

```diff
@@ -18,3 +20,9 @@ body = """
 {% endfor -%}
 {% endfor -%}
 """
+
+[[package]]
```

Comment:
why this?

Reply:
there has to be one package that gets tagged per the docs linked

Reply:
yea we talked about this IRL, forgot to resolve comment